build-parent/pom.xml

```diff
@@ -122,6 +122,8 @@
     <version.smallrye.jwt>1.1.0</version.smallrye.jwt>
     <version.smallrye-fault-tolerance>1.0.1</version.smallrye-fault-tolerance>
 
+    <!-- For PME alignments -->
```

Comment:
Why do we need to align slf4j? What's bringing in that dependency?

My concern is it will have knock-on effects with our dependencies when we're not really meant to be dictating a particular logging framework

Reply:
I don't recall right now, there were various minor variations, an this is what was set in the product branch in the previous product. But it will make sense to deal with the specific version updates at the build parent level in a separate PR